src/main/java/com/example/controller/SimulationController.java

```diff
@@ -45,34 +45,34 @@ public class SimulationController {
     private IntegerSettingTextField q;
     @FXML
     private IntegerSettingTextField time;
-    @FXML
-    private Button startButton;
-    @FXML
-    private Button nextStepButton;
-    @FXML
-    private Button liveButton;
-    @FXML
-    private Button instantFinishButton;
-    @FXML
-    private Button pauseButton;
+
     @FXML
     private ComboBox<AlgorithmType> algorithmsBox;
     @FXML
     private Slider animationSpeedSlider;
 
-    private AlgorithmSettings algorithmSettings = new AlgorithmSettings();
+    private final AlgorithmSettings algorithmSettings = new AlgorithmSettings();
 
     private final Map<AlgorithmType, List<Node>> options = new HashMap<>();
 
-    private ObservableList<AlgorithmType> availableAlgorithms = FXCollections.emptyObservableList();
+    @Getter
+    private final BooleanProperty startProperty = new SimpleBooleanProperty();
```

Comment:
Names of these properties indicates that they inform us about possibility to start, do next step etc. but in fact they mean something exactly different. I know that they are constructed like this, because javafx buttons don't have enableProperty, however I would either negate these properties in disabledProperty() or rename them.